pkg/acl/access_control_list_middleware_test.go

```diff
@@ -95,20 +114,21 @@ func Test_AccessControlListMiddleware_UserHasNoAccess(t *testing.T) {
 			if err != nil {
 				t.Fatal(err)
 			}
```

Comment:
It is not part of this PR, but can we change this to 
```go
Expect(err).NotTo(BeOccurred())
```

Reply:
changed that. thanks